boost-gradle/src/main/groovy/io/openliberty/boost/gradle/utils/GradleProjectUtil.groovy

```diff
@@ -37,4 +40,33 @@ public class GradleProjectUtil {
 
         return version        
     }
+    
+    public static List<String> getSpringFrameworkDependencies(Project project) {
+
+        List<String> springBootDependencies = new ArrayList<String>()
+
+        project.configurations.compile.resolvedConfiguration.resolvedArtifacts.collect { it.moduleVersion.id }.each { ModuleVersionIdentifier id ->
+        
+            if ( id.group.equals("org.springframework") ) {
+                
+                springBootDependencies.add(id.name.toString())
+            }
+        }
+       
+        return springBootDependencies
+    }
+    
+    public static Map<String, String> getBoosterDependencies(Project project) {
+        Map<String, String> boosterDependencies = new HashMap<String, String>()
+
+        project.configurations.compile.resolvedConfiguration.resolvedArtifacts.collect { it.moduleVersion.id }.each { ModuleVersionIdentifier id ->
```

Comment:
Does this work the same as the ArtifactResolutionQuery we were using before in BoostPackageTask? That was being used to find all of the poms in the project's dependency tree.

Reply:
I'm fairly certain this is getting all of the dependencies in the poms. I will print them out using both ways and make sure.

Reply:
Did a bit of testing. Looks good.

Reply:
Thanks Matt